yarn.lock

```diff
@@ -2,67 +2,13 @@
 # yarn lockfile v1
 
 
-"@47ng/cloak@^1.1.0":
```

Comment:
yarn.lock seems to have changed a lot. Are we sure there is nothing else here apart from og-image deps.

Reply:
Oh you're right. Overlooked that. I'll run yarn again in a sec. Cloned stuff from scratch, that's probably why there's a lot of changes.

Reply:
Nope. Even after resetting the lock file and only updating `@vercel/og` it does apply **all** these changes in the yarn.lock. Weird. I guess the lock file simply hasn't been committed for a long time? 🤔

Reply:
Yeah it's possible.